calico.go

```diff
@@ -306,6 +307,33 @@ func cmdDel(args *skel.CmdArgs) error {
 		"Node":         nodename,
 	}).Info("Extracted identifiers")
 
+	calicoClient, err := CreateClient(conf)
+	if err != nil {
+		return err
+	}
+
+	ep := api.WorkloadEndpointMetadata{
+		Name:             args.IfName,
+		Node:             nodename,
+		ActiveInstanceID: args.ContainerID,
+		Orchestrator:     orchestrator,
+		Workload:         workload,
+	}
+
+	wep, err := calicoClient.WorkloadEndpoints().Get(ep)
```

Comment:
Does this `Get()` require the ActiveInstanceID to be present? What happens if it's missing (e.g. on upgrade?)